dictionary/MessageFormatDictionary.js

```diff
@@ -97,6 +97,31 @@ export default class MessageFormatDictionary extends Dictionary {
 
 		return scope(parameters);
 	}
+
+	/**
+	 * @inheritdoc
+	 * @param {string} key The key identifying the localization phrase. The key
+	 *        consists of at least two parts separated by dots. The first part
+	 *        denotes the name of the source JSON localization file, while the
+	 *        rest denote a field path within the localization object within
+	 *        the given localization file.
+	 */
+	has(key) {
+		let path = key.split('.');
+		let scope = this._dictionary;
+
+		for (let scopeKey of path) {
+			if (!scope[scopeKey]) {
+				// not exists
```

Comment:
This is an unnecessary comment that just repeats what is already stated in the docblock (and pretty obvious from the method's name). Please remove it.